Makefile

```
@@ -24,5 +25,6 @@ build:
 test:
 	@docker build --target=test -t datamegh:test .
 	@docker run datamegh:test
+	@pyright
```

If you add the script `typecheck` in the `package.json` then you can just do like this:

```suggestion
	@npm run typecheck
```

---

Yes. Thank you for suggestion